frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,817 @@
+                                alt={`Thumbnail ${index + 1}`}
+                                className="w-full h-full object-cover"
+                            />
+                        </button>
+                    ))}
+                </div>
+            )}
+
+            {/* Full Screen Modal */}
+            {isModalOpen && (
+            <div 
+                role="dialog"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<dialog>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)